### PR TITLE
Add missing recipes, add cobblestone walls

### PR DESCRIPTION
**Pull request by mounderfod**

### CODE REVIEW

File: src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    }
+  },
+  "result": {
+    "item": "proton:cobblestone_bricks",
+    "count": 4
+  },
+  "conditions": [
+    {
+      "type": "proton:flag",
+      "flag": "cobblestone_bricks"
```
Comment (YTG1234) [inline src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json:19]: Where is this flag defined?

**Review by YTG1234 — APPROVED** [review]

File: src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json (hunk 1 above)
Comment (mounderfod) [reply src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json:19]: I believe it depends on the module configs.

Comment (YTG1234) [reply src/main/resources/data/proton/recipes/cobblestone_bricks/cobblestone_bricks.json:19]: I don't see it defined anywhere though. What mod adds `"conditions"` anyway?

**Review by namebee — CHANGES_REQUESTED** [review]
I am not accepting this, We're almost ready to dynamically generate recipes. This is just unnecesary bloat.

**Comment by mounderfod** [comment]
:concern:

**Comment by mounderfod** [comment]
i'm out of patience at this point

**Comment by mounderfod** [comment]
my previous files were deleted by merging, and I see NO reason why this should be denied.

**Pull request closed without merging**